lib/components/src/docsEscapeHatch.ts

```diff
@@ -0,0 +1,22 @@
+import { FunctionComponent } from 'react';
+
+function caml(name: string): string {
+  return name
+    .replace(/([a-z])([A-Z])/g, '$1-$2')
+    .replace(/[\s_]+/g, '-')
+    .toLowerCase();
+}
+
+function additionalClassesFormatting(additionalClasses: string): string {
+  return additionalClasses ? ` ${additionalClasses}` : '';
+}
+
+export function docsEscapeHatch(component: FunctionComponent, additionalClasses?: string): string {
+  const name = component.displayName || component.name || '';
+  const cleanName = `sbdocs sbdocs-${caml(name)}${additionalClassesFormatting(additionalClasses)}`;
```

Comment:
What if name is empty?

The whole point of accepting additionalClasses is to output the proper formatting when we can't find a name for the component.

Reply:
`docsEscapeHatch` should call `docsEscapeHatchFromId`